modules/videoio/src/cap_ffmpeg.cpp

```diff
@@ -112,7 +112,7 @@ class CvCapture_FFMPEG_proxy CV_FINAL : public cv::IVideoCapture
         }
 
         cv::Mat tmp(height, width, CV_MAKETYPE(CV_8U, cn), data, step);
-        this->rotateFrame(tmp);
+        applyMetadataRotation(*this, tmp);
```

Comment:
Apparently this PR breaks FFmpeg plugin build (with `-DVIDEOIO_PLUGIN_LIST=ffmpeg` option). In this mode it does not compile `cap.cpp` which contains `applyMetadataRotation` function and can not find it. Perhaps we could move this function to one of headers used by both AVFoundation and FFmpeg backends or to a separate source module and add it to the plugin compilation process.

Reply:
Should I take care of this in a separate PR or will you do it internally in the team?

Reply:
I'll try to fix it today.